src/layers/word2vec_reader.h

```diff
@@ -18,8 +18,6 @@ class Word2VecReader {
     LOG(info, "[data] Loading embedding vectors from {}", fileName);
 
     io::InputFileStream embFile(fileName);
-    ABORT_IF(!embFile.isOpen(),
-             "Unable to open file with embeddings: " + fileName);
```

Comment:
We don't need this or the condition is checked implicitly by InputFileStream?

Reply:
We are now aborting on badbit and failbit. This was not working before. So a failure to read aborts anyway.